includes/defaults.inc.php

```diff
@@ -712,6 +712,7 @@
 $config['poller_modules']['ntp']                         = 1;
 $config['poller_modules']['services']                    = 1;
 $config['poller_modules']['loadbalancers']               = 0;
+$config['poller_modules']['mef-evc']                     = 0;
```

Comment:
mef-evc seems very generic, should we do this as just mef or is that to generic?